controllers/ProfileController.js

```diff
@@ -33,26 +29,18 @@ class ProfileController {
           message = 'username is';
         }
       } else if (email) {
-        const emailOwner = await User.findOne({
-          where: { email }
-        });
+        const emailOwner = await User.findOne({ where: { email } });
         if (emailOwner) {
           message = 'email is';
         }
       } else if (username) {
-        const usernameOwner = await User.findOne({
-          where: { username }
-        });
+        const usernameOwner = await User.findOne({ where: { username } });
         if (usernameOwner) {
           message = 'username is';
         }
       }
       if (message) {
-        return res.status(409).send({
-          errors: {
-            body: [`${message} already taken`]
-          }
-        });
+        return res.status(409).send({ errors: { body: [`${message} already taken`] } });
```

Comment:
@NkFab The status code in your response body is required

Reply:
@oesukam This is not in the file I'm working on.

Reply:
who's file is it?

Reply:
@NkFab Git shows that you made some changes, that is the reason I commented on it

Reply:
It is Daniel's, maybe it is showing that because of a setting I changed in es-lint but I never touched that code.